src/ReinforcementLearningZoo/src/algorithms/policy_gradient/sac.jl

```diff
@@ -119,13 +119,18 @@ function (p::SACPolicy)(env)
     p.update_step += 1
 
     if p.update_step <= p.start_steps
-        p.start_policy(env)
+        action = p.start_policy(env)
+        if(size(action[1]) != ())
+            action = permutedims(hcat(action...))
```

Comment:
```suggestion
            action = permutedims(reduce(hcat, action))
```
In Julia (you mentioned you are new to the language) you want to avoid using the splat operator to do this kind of stuff because it's really slow. `reduce(hcat, yourvectorofvectors)` does that much faster.

I also want to ask, why are the dimensions permuted afterward? If `action` is a collection of vectors, your action matrix will have actions as rows instead of columns. If you come from python you should know that Julia matrices are "column-major". So if this is why you used permutedims, it should also be removed.

Reply:
I found that I had to reshape the output of the RandomPolicy, since it gives me a vector of vectors instead of a matrix. Thinking about it, this might indeed be an issue if one wants to use any other start_policy.
There might just be a better way to generate the random actions in the beginning for multiple envs. I guess the issue is, that a RandomPolicy will not adjust if it is working on a MultiThreadedEnv and my solution of using a Space(Space(...)) in the RP will give me those Vectors. Do you maybe have a suggestion on how to setup the RandomPolicy for MutliThreadedEnvs? That would basically make the whole if statement unnecessary.

Reply:
Hi,

I'm not sure I understand your message. Basically what I mean regarding permutedims is this : 
![image](https://user-images.githubusercontent.com/47037088/178712935-cf961d95-e830-445c-ae02-3e65272c3540.png)
The actions should be the columns of the resulting matrix, whereas permutedims leads to row actions. I think the if statement is fine. I haven't dealt with MultiThreadedEnvs enough to investigate this right now but if this remains a problem later I can check it out

Reply:
Can you perhaps show me a screenshot of this problem?

Reply:
I started setting up an example when I found an easy solution:
I just had to switch around the 2 Spaces inside the RandomPolicy definition (basically na <--> N_ENV). Then i dont need the permutedims. Not sure why I never tried that. Its more intuitive to me this way around anyways, so obviously the better solution.
I updated the experiment in my first comment and pushed the better version to my branch.
Thanks for helping me understand how that stuff works.

Reply:
Perfect!